src/main/kotlin/com/terraformation/backend/customer/model/FunderUser.kt

```diff
@@ -0,0 +1,78 @@
+package com.terraformation.backend.customer.model
+
+import com.terraformation.backend.auth.CurrentUserHolder
+import com.terraformation.backend.db.default_schema.FacilityId
+import com.terraformation.backend.db.default_schema.GlobalRole
+import com.terraformation.backend.db.default_schema.OrganizationId
+import com.terraformation.backend.db.default_schema.Role
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.default_schema.UserType
+import com.terraformation.backend.log.perClassLogger
+import java.time.Instant
+import java.time.ZoneId
+import java.time.ZoneOffset
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.userdetails.UserDetails
+
+class FunderUser(
+    val createdTime: Instant,
+    override val userId: UserId,
+    override val authId: String?,
+    val email: String,
+    val emailNotificationsEnabled: Boolean,
+    val firstName: String?,
+    val lastName: String?,
+    val cookiesConsented: Boolean?,
+    val cookiesConsentedTime: Instant?,
+) : TerrawareUser, UserDetails {
+  companion object {
+    private val log = perClassLogger()
+  }
+
+  override val timeZone: ZoneId
+    get() = ZoneOffset.UTC
```

Comment:
Is this right? I'd think we'd want to let funder users set their time zones like we do for regular users, but maybe not.

Reply:
Will check with product and work on understanding more about how current users set their timezones.

Reply:
Product confirmed adding all the same settings to Funders. Michael will likely update the design to reflect so I'll get this file updated to reflect as well. Thanks for calling this out!